### DISCUSSION TITLE
How to remove the www.ventoy.net in GRUB menu

### DISCUSSION THREAD

**Discussion by anastaciovince** [ventoy/Ventoy#2656] [Q&A] [answered]
![image](https://github.com/ventoy/Ventoy/assets/59636944/88d0d9a1-43ad-4712-945d-f85a1361e9cb)

**Comment by steve6375** [comment, marked as answer]
The whole string can be easily removed from any theme. The secret is in the eBook 'Getting started with Ventoy' but you cannot publish the solution on any forum or internet because it is copyrighted information which is only available in the eBook ;-)

**Reply by anastaciovince** [reply]
hello, I already did it
From my screenshot it says I can change the color of the text
so I just change it same with the color of the background, making the text invisible
thank you

**Reply by steve6375** [reply]
There book describes a better and easier way. No colour matching is required.
